NEWS.md

```diff
@@ -36,6 +36,8 @@
 
 5. `fwrite(x, row.names=TRUE)` with `x` a `matrix` writes `row.names` when present, not row numbers, [#5315](https://github.com/Rdatatable/data.table/issues/5315). Thanks to @Liripo for the report, and @ben-schwen for the fix.
 
+6. `melt` with a list for `measure.vars` was inconsistent between length=1 list and length>1 list, [#5209](https://github.com/Rdatatable/data.table/issues/5209). Thanks to @tdhock for reporting and fixing.
```

Comment:
Note: We usually write NEWS items in terms of the new behavior, not in terms of what used to be a problem, e.g. here "melt... is consistent for any length of `list()` input to `measure.vars`". It might also be prudent to describe what is now consistent, i.e. "melt returns an `integer` column for `variable` whenever `measure.vars` is a `list()`, consistent with the documented behavior".

Reply:
sounds great, thanks for the suggestion.